relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics.snap

```diff
@@ -604,7 +604,9 @@ expression: metrics
             "http.status_code": "500",
             "span.action": "INSERT",
             "span.category": "db",
+            "span.description": "INSERT INTO from_date (..) VALUES (%s)",
```

Comment:
Are we using `(..)` on replacements, or just `..`? Can this result in unexpected aggregation outcomes?